lib/PayPal/Core/PayPalHttpConfig.php

```diff
@@ -229,9 +234,10 @@ public function setHttpTimeout($timeout)
      * Set HTTP proxy information
      *
      * @param string $proxy
+     * @param array $otherConfigs you can set other proxy configurations
      * @throws PayPalConfigurationException
      */
-    public function setHttpProxy($proxy)
+    public function setHttpProxy($proxy,$otherConfigs=[CURLOPT_PROXYAUTH=>CURLAUTH_NTLM,CURLOPT_HTTPPROXYTUNNEL=>1,CURLOPT_SSL_VERIFYPEER=>0])
```

Comment:
We can move these $otherConfigs to something similar to [$defaultCurlOptions](https://github.com/laurovalente/PayPal-PHP-SDK/blob/master/lib/PayPal/Core/PayPalHttpConfig.php#L21)
such as `$defaultProxyCurlOptions`. This should allow developers to update them from the outside.